src/GTP.cpp

```diff
@@ -686,6 +687,9 @@ bool GTP::execute(GameState & game, std::string xinput) {
             gtp_fail_printf(id, "cannot load file");
         }
         return true;
+    } else if (command.find("reiterate_moves") == 0) {
+        gtp_printf(id, game.reiterate_moves().c_str());
```

Comment:
The discussion in the topic gives some suggestions to avoid this. I think this is a really bad command to add, because obviously printsgf/loadsgf already allow recovering moves, so this is duplicate functionality.